src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -334,4 +345,33 @@ private static void setInstanceEdits(int personId) throws Exception
 		EWASInferrer.setInstanceEdit(instanceEditInst);
 		PathwaysInferrer.setInstanceEdit(instanceEditInst);
 	}
+
+	/**
+	 * Download the Wormbase file that contains the WBGene IDs mapped to gene names, and then create a mapping of IDs to names.
```

Comment:
Maybe just add a note that the file will _only_ be downloaded if it doesn't already exist. Also, is there any mechanism to clean up old files to ensure fresh files are downloaded?

Reply:
Joel raised a good point that since the file isn't all that big that the file should just be redownloaded each time

Reply:
Oh, ok then.

Reply:
File is now always downloaded, and then deleted once it creates the mappings